src/ScheduledTasks/Logging/Logs.Entity.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Log messages.
+/// </summary>
+/// <remarks>
+/// NOTE: Trying to make logs consistent with https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/Logging/LogEvents.cs.
```

Comment:
consistent in what way?

Reply:
need some advice on this part, consistent with durabletask.core logging eventids. i checked it range from 0 to 80, and put my new loggings after it

Reply:
are these eventid ranges look right or there are specific ranges i should put into

Reply:
I am not sure if there is a requirement on ranges being distinct, and what should be done in this context. Maybe @cgillum has some answers.

Reply:
At one point I had an idea that using consistent event IDs could be beneficial in cases where we wanted to write Kusto queries that were consistent across different SDKs, but over time I've realized that it's quite hard to maintain this, and the benefit is very unclear. I'd suggest not worrying about maintaining consistency.

Reply:
Yeah, I think we are making it harder for ourselves. I think we can even let the source generator determine these event IDs?